recipes/trailrunner/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "trailrunner" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trailrunner-{{ version }}.tar.gz
+  sha256: 157de2b2d82139d9583ec1dbc1981c33b39e24c420bb5294f86083f1f4b44311
+  patches:
+    # Building of the package fails as FLIT tries to determine the package version
+    # by importing the module. This fails as dependencies are not yet installed.
+    # Include the version directly in the top-level module to avoid this check.
+    - 0001-fix_flit_inline_package_version.patch
```

Comment:
Oh! You don't need to patch it like this b/c it will be a nightmare to maintain. In pathological cases where the package imports itself to get the version we are OK having the run dependencies in host/build. Better to just open an issue upstream and wait for them to fix it.

Reply:
I reverted to adding the required runtime dependencies to build time and created an upstream bug: https://github.com/omnilib/ufmt/issues/123